Kernel/Bus/PCI/Device.h

```diff
@@ -35,4 +40,14 @@ class Device {
     Address m_pci_address;
 };
 
+template<typename... Parameters>
+void dbesgln_pci(Device const& device, AK::CheckedFormatString<Parameters...>&& fmt, Parameters const&... parameters)
+{
+    AK::StringBuilder builder;
+    builder.append("{}: {}: "sv);
+    builder.append(fmt.view());
```

Comment:
This is even worse, now we don't even know that this code can fail at runtime due to OOM. We should not be using the non-failable APIs anymore. In the previous comment by Brian, he mentioned it is a bad idea to use `MUST`, because it will crash the kernel. So instead, what he suggested is to make the `dmesgln_pci` function failable too, by changing the signature to `ErrorOr<void>`.

Reply:
Ah I see - I misunderstood. We want to have the caller handle the error by using TRY instead?

Reply:
Yes, that would be better indeed! Eventually we want to be able to handle all the OOM cases in the Kernel, such that we never panic the Kernel. So, starting this change here would be appreciated! You can add FIXMEs if there is no way to propagate the error at the callsite (by using `release_value_but_fixme_should_propagate_errors` for example).